packages/stark-ui/assets/styles/_header.scss

```diff
@@ -103,6 +88,8 @@
         display: flex;
         justify-content: center;
         width: 34%;
+        align-self: center;
+        margin-bottom: 15px;
```

Comment:
The same as for my previous remark, we should not add this rules in `stark-ui`, they should be in the showcase instead...